scripts/globals/mobskills/berserk.lua

```diff
@@ -13,10 +13,19 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    local typeEffect = xi.effect.BERSERK
     local power = (116 / 256) * 100
-    skill:setMsg(xi.mobskills.mobBuffMove(mob, typeEffect, power, 0, 180))
-    return typeEffect
+
+    if
+        mob:getFamily() == 83 or
```

Comment:
might be worth commenting which families these are

Reply:
In ~future time we should add getSuperFamilyName() since I think all of these are just things that happen because two superfamilies use moves with the same name